integration-tests/src/main/java/org/apache/polaris/service/it/test/PolarisManagementServiceIntegrationTest.java

```diff
@@ -546,11 +403,10 @@ public void testCreateCatalogWithoutStorageConfig() throws JsonProcessingExcepti
   }
 
   @Test
-  public void testCreateCatalogWithUnparsableJson() throws JsonProcessingException {
+  public void testCreateCatalogWithUnparsableJson() {
     String catalogString = "{\"catalog\": {{\"bad data}";
     try (Response response =
-        newRequest("http://localhost:%d/api/management/v1/catalogs", userToken)
-            .post(Entity.json(catalogString))) {
+        managementApi.request("v1/catalogs").post(Entity.json(catalogString))) {
       assertThat(response)
           .returns(Response.Status.BAD_REQUEST.getStatusCode(), Response::getStatus);
       ErrorResponse error = response.readEntity(ErrorResponse.class);
```

Comment:
The error message is not normative; I think it should not be checked. For Quarkus, the message is `HTTP 400 Bad Request`.

Reply:
And BTW, this is the only test still failing with the Quarkus runtime.

Reply:
#667

Reply:
rebased